source/features/collapse-markdown-sections.css

```diff
@@ -0,0 +1,4 @@
+/* Remove the default triangle marker and the padding */
+.rgh-markdown-section > summary {
+	display: block !important;
+}
```

Comment:
Chrome, doesn't work:

<img width="908" alt="Screen Shot 5" src="https://user-images.githubusercontent.com/1402241/109395835-171c0e00-78f4-11eb-8225-820c2dd42361.png">

This is why I suggested _not_ using details/summary. Since we need the listener for the alt-click, we might as well use the listener to toggle content.

This has also the advantage of not having to touch the DOM at all until the user needs the feature.